polkadot/node/core/pvf/prepare-worker/src/lib.rs

```diff
@@ -331,7 +382,7 @@ struct JobResponse {
 /// - If success, pipe back `JobResponse`.
 fn handle_child_process(
 	pvf: PvfPrepData,
-	mut pipe_write: PipeWriter,
+	pipe_write: &mut PipeWriter,
```

Comment:
I'm curious, why this change?

Reply:
I just tested and I think it compiled fine with `mut pipe_write` (how it was before).